### PR TITLE
Synergy Casks added

### PR DESCRIPTION
**Pull request by k0ssi**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/synergy.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version '1.7.6'
+  sha256 '838c14f253bed25ac6cf4b1887bb0cdae530ea969fbe53112aa4dd480e93887a'
+
+  url 'http://symless.com/files/packages/synergy-v1.7.6-stable-bcb9da8-MacOSX1011-x86_64.dmg?v32'
+  name 'Synergy'
+  homepage 'http://symless.com/'
+  license :commercial
+
+  app 'Synergy.app'
+  uninstall delete: '/Applications/Synergy.app'
```
Comment (vitorgalvao) [inline Casks/synergy.rb]: Remove this. Not necessary.

File: Casks/synergy.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'synergy' do
+  version '1.7.6'
+  sha256 '838c14f253bed25ac6cf4b1887bb0cdae530ea969fbe53112aa4dd480e93887a'
+
+  url 'http://symless.com/files/packages/synergy-v1.7.6-stable-bcb9da8-MacOSX1011-x86_64.dmg?v32'
+  name 'Synergy'
+  homepage 'http://symless.com/'
```
Comment (vitorgalvao) [inline Casks/synergy.rb]: `homepage 'http://symless.com/synergy/'`

File: Casks/synergy.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'synergy' do
+  version '1.7.6'
+  sha256 '838c14f253bed25ac6cf4b1887bb0cdae530ea969fbe53112aa4dd480e93887a'
+
+  url 'http://symless.com/files/packages/synergy-v1.7.6-stable-bcb9da8-MacOSX1011-x86_64.dmg?v32'
```
Comment (vitorgalvao) [inline Casks/synergy.rb]: `url "http://symless.com/files/packages/synergy-v#{version}-stable-bcb9da8-MacOSX1011-x86_64.dmg"`

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ea70191faa2f1a427648c71b26a1b7532397ff30. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**